Core/Src/bmbInterface.c

```diff
@@ -549,11 +572,15 @@ void resetASCI()
 */
 bool initASCI()
 {
+	Debug("Initializing ASCI connection...\n");
 	resetASCI();
 	csOff();
 	bool successfulConfig = true;
 	// dummy transaction since this chip sucks
 	readRegister(R_CONFIG_3);
+	readRegister(R_CONFIG_3);
```

Comment:
TODO: Determine if all of these transactions are actually required or if we just need the one